apps/src/applab/designMode.js

```diff
@@ -840,7 +840,7 @@ function makeDraggable(jqueryElements) {
         });
 
         // Dim the element if it's dragged out of bounds
-        if (!isMouseInBounds(newLeft, newTop)) {
+        if (!isMouseInBounds(newLeft + event.offsetX, newTop + event.offsetY)) {
```

Comment:
Can the cursor position actually be (slightly) incorrect in this check since we snap `newLeft` and `newTop` to the grid [just a few lines above](https://github.com/code-dot-org/code-dot-org/pull/12519/files#diff-7f9b30159330297b3b2915dcafb93087R829)?  I don't see where the event offset would be updated to account for that possible late change in the element position.  I might be imagining it, but I think I can see this in your screencast.  I'll have to pull and build to verify it.